plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/maven2/Maven2Format.java

```diff
@@ -41,6 +47,37 @@
   public static final String METADATA_CONTENT_TYPE = ContentTypes.TEXT_XML;
 
   /**
+   * {@link TimeZone} of timestamps used in Maven2 repository metadata.
+   */
+  public static final TimeZone METADATA_TIMESTAMP_UTC_TZ = TimeZone.getTimeZone("UTC");
+
+  /**
+   * {@link Supplier} of dotted {@link DateFormat} for timestamps used in Maven2 repository metadata.
+   */
+  public static final Supplier<DateFormat> METADATA_DOTTED_TIMESTAMP_SUPPLIER = new Supplier<DateFormat>()
```

Comment:
I'd prefer to use joda-time here if possible as it has thread-safe formatters (we use them in the NuGet plugin)